apps/portal/prisma/new-schema.prisma

```diff
@@ -0,0 +1,186 @@
+    eventId   String
+    status    String
+
+    @@unique([profileId, eventId])
+    @@map("event_reservations")
+}
+
+model Session {
+    id           String   @id @default(auto()) @map("_id") @db.ObjectId
+    sessionToken String   @unique
+    userId       String
+    expires      DateTime
+    user         User     @relation(fields: [userId], references: [id], onDelete: Cascade)
+
+    @@map("sessions")
+}
+
+model Role {
+    id       String @id @default(auto()) @map("_id") @db.ObjectId
+    roleName String
+
+    RolesOnUser RolesOnUser[]
+
+    @@map("roles")
+}
+
+model RolesOnUser {
+    id        String   @id @default(auto()) @map("_id") @db.ObjectId
+    role      Role     @relation(fields: [roleId], references: [id])
+    roleId    String
+    user      User     @relation(fields: [userId], references: [id])
+    userId    String
+    createdAt DateTime @default(now())
+
+    @@unique([userId, roleId])
+}
+
+model User {
+    id            String        @id @default(auto()) @map("_id") @db.ObjectId
+    name          String?
+    email         String?       @unique
+    emailVerified DateTime?
+    image         String?
+    accounts      Account[]
+    sessions      Session[]
+    profile       Profile?
+    roles         RolesOnUser[]
+
+    user_app UserAppJunction @relation(fields: [userId], references: [id])
+    @@map("users")
+}
+
+model VerificationToken {
+    id         String   @id @default(auto()) @map("_id") @db.ObjectId
+    identifier String
+    token      String   @unique
+    expires    DateTime
+
+    @@unique([identifier, token])
+}
+
+model Profile {
+    id               String    @id @default(auto()) @map("_id") @db.ObjectId
+    firstName        String
+    lastName         String
+    email            String    @unique
+    netid            String    @unique
+    classStanding    String
+    major            String
+    utdStudent       Boolean
+    roles            String[]
+    membershipStatus Boolean
+    membershipTS     DateTime?
+    resume           Boolean
+    resumeTS         DateTime?
+
+    user User @relation(fields: [userId], references: [id])
+    officer Officer @relation(fields: [profileId], references: [id])
+
+    userId String             @unique @db.ObjectId
+    events EventReservation[]
+}
+
+model Event {
+    id          String             @id @default(auto()) @map("_id") @db.ObjectId
+    summary     String
+    description String
+    url         String
+    location    String
+    start       DateTime
+    end         DateTime
+    profiles    EventReservation[]
+
+    @@map("events")
+}
+
+model TypeformApplication {
+    id                  String  @id @default(auto()) @map("_id") @db.ObjectId
+    active              Boolean
+    description         String
+    endpoint            String
+    externalResourceUrl String
+    typeformId          String
+    typeformName        String
+
+    user_app UserAppJunction @relation(fields: [formmId], references: [typeformId])
```

Comment:
Remove this because Portal does not keep track of response data for Typeform application. There should be a separate Application object for the new app system.

Reply:
Ok - for right now, ive replaced that entirely with a new Application object

Reply:
Might be better to keep both `TypeformApplication` and `Application` until we're 100% sure that the new app system is functional (just in case if something goes wrong, we can revert back to `TypeformApplication`)